stew/bitops2.nim

```diff
@@ -344,6 +346,11 @@ func log2trunc*(x: SomeUnsignedInt): int {.inline.} =
       else:
         log2truncNim(x)
 
+template bitWidth*(x: SomeUnsignedInt): int =
```

Comment:
hm, I think this has an established name somewhere.. https://en.wikipedia.org/wiki/Find_first_set maybe has a hint (else I'll dig later)